azure-pipelines/templates/osx-build.yml

```diff
@@ -1,5 +1,5 @@
 steps:
-- script: './configure PKG_CONFIG_PATH=/usr/local/lib/pkgconfig --enable-static'
+- script: 'MACOSX_DEPLOYMENT_TARGET=10.9 ./configure PKG_CONFIG_PATH=/usr/local/lib/pkgconfig --enable-static'
```

Comment:
Please put this in a parameters (see `windows-build.yml`). That way we can sync the '10.9' value between the files and instances. Than in `azure-pipelines-ci.yml` you can set the parameter for both `build` and `dependencies`; easier to spot they are equal. I think.